utils/installer/config.example.lua

```diff
@@ -128,58 +74,40 @@ lvim.builtin.treesitter.highlight.enable = true
 --   buf_set_option("omnifunc", "v:lua.vim.lsp.omnifunc")
 -- end
 
--- -- set a formatter, this will override the language server formatting capabilities (if it exists)
+-- -- linters and formatters <https://www.lunarvim.org/docs/languages#lintingformatting>
 -- local formatters = require "lvim.lsp.null-ls.formatters"
 -- formatters.setup {
---   { command = "black", filetypes = { "python" } },
---   { command = "isort", filetypes = { "python" } },
+--   { name = "stylua" },
```

Comment:
the `name` key is kept for backwards compatibility, since `command` matches what null-ls uses internally

```suggestion
--   { command = "stylua" },
```